src/index.js

```diff
@@ -33,8 +23,60 @@ items.forEach((item) => {
 
   input.addEventListener('change', () => {
     updateCompleted(item, input);
-    saveLocalStorage(items);
+    saveLocalStorage(itemsArray);
   });
 
   todos.appendChild(div);
+  const vertical = div.querySelector('.vertical');
+  const trash = div.querySelector('.trash');
+  const edit = div.querySelector('.edit');
+  edit.addEventListener('change', () => {
+    itemsArray.forEach((item) => {
+      if (item.index.toString() === div.id) {
+        item.description = edit.value;
+      }
+    });
+    saveLocalStorage(itemsArray);
+  });
+  vertical.addEventListener('click', () => {
+    vertical.style.display = 'none';
+    trash.style.display = 'block';
+  });
+  trash.addEventListener('click', () => {
+    div.remove();
+    itemsArray = itemsArray.filter((something) => {
+      if (something.index.toString() === div.id) {
+        return false;
+      }
+      return true;
+    });
+    saveLocalStorage(itemsArray);
+  });
+});
```

Comment:
- [ ] It would be so much better if all of the actions for add,  delete, update, and clear all are in their own file. Please move the actions for the CRUD into another file and imported them as modules. As is one of the requirements and this helps you to refactor and maintain your code easier. 
- [ ]  The update action is not updating the UI, The update is reflected in the local storage but once refresh disappear from the page. 

![Capture](https://user-images.githubusercontent.com/64173189/134750261-f0db4633-59d2-4c0c-916c-81e1318d30bb.JPG)

🛠️ 👍